.ipynb_checkpoints/command_analogues/cat.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument('path', nargs='*', help='files to read', type=str)
+
+args = parser.parse_args()
+
+if len(args.path)==0 or args.path[0]=='-':
+	while True:
+		x=sys.stdin.readline()
+		sys.stdout.write(x)
+else:
+	for object in args.path:
+		if  os.path.isfile(object):
+			with open(object) as file:
+				for line in file.readlines():
+					sys.stdout.write(line)
```

Comment:
Можно даже чуть проще
```suggestion
			sys.stdout.write(file.read())
```